deploy/Dockerfile

```diff
@@ -25,24 +25,28 @@ RUN BUILD_ONLY=true yarn install \
 
 COPY code-layer/ ./
 
-# Setup PaperTrail logging, note that the port 41828 is
-# a port that has been assigned for Cardstack's account.
+# Uncomment to set up PaperTrail logging.
+# This service helps you easily access Docker container logs.
+# Note that the port number may need to be configured for new accounts.
 # Log file emitters can be managed in PaperTrail's dashboard.
-RUN curl -L https://github.com/papertrail/remote_syslog2/releases/download/v0.20/remote-syslog2_0.20_amd64.deb -o remote-syslog2.deb \
-  && dpkg -i ./remote-syslog2.deb \
-  && apt-get install -f \
-  && rm ./remote-syslog2.deb
-RUN mkdir -p /srv/hub/logs \
-  && echo '\
-files: \n\
-  - /srv/hub/logs/node.log\n\
-destination:\n\
-  host: logs3.papertrailapp.com\n\
-  port: 41828\n\
-  protocol: tls\n\
-pid_file: /var/run/remote_syslog.pid'\
-> /etc/log_files.yml
+# RUN curl -L https://github.com/papertrail/remote_syslog2/releases/download/v0.20/remote-syslog2_0.20_amd64.deb -o remote-syslog2.deb \
+#  && dpkg -i ./remote-syslog2.deb \
+#  && apt-get install -f \
+#  && rm ./remote-syslog2.deb
+# RUN mkdir -p /srv/hub/logs \
+# && echo '\
+# files: \n\
+#  - /srv/hub/logs/node.log\n\
+# destination:\n\
+#  host: logs3.papertrailapp.com\n\
+#  port: 41828\n\
```

Comment:
this port is specific to our papertrail account. let's mask this port number like `XXXXX` so we dont have people inadvertently emitting logs to our papertrail if they uncomment this.

Reply:
The port is changed to `XXXXX` and I clarified the configuration in the comment.